### PR TITLE
docs: Add troubleshooting note for neo4j container

### PR DESCRIPTION
**Pull request by prafullkotecha**
### Summary of Changes

- Added troubleshooting note, for neo4j container, to Installation instructions.
- Added image file, used by the screenshot, to `img` directory
- Updated `.gitignore` file to ignore `.vscode` files

### Documentation

Added error logs from neo4j container, and screenshot of VS Code showing how to change the setting. End of Line sequence in `neo4j.conf` file should be `LF`, instead of `CRLF`, at least on `Windows` OS.

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [x] PR title addresses the issue accurately and concisely, including a title prefix.
- [x] PR includes a summary of changes.
- [x] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

**Comment by prafullkotecha** [comment]
@feng-tao @Golodhros - Can you please review this minor change to the documentation. Let me know if it needs to be improved in any way.

**Review by Golodhros — APPROVED** [review]
LGTM

File: docs/installation.md [hunk 1]
```diff
@@ -76,6 +76,28 @@ Atlas would be ready once you'll have the following output in the docker output
...
+neo4j_amundsen      |   data:         /neo4j/data
+neo4j_amundsen      |   certificates: /var/lib/neo4j/certificates
+neo4j_amundsen      |   run:          /var/lib/neo4j/run
+neo4j_amundsen      | Starting Neo4j.
+'eo4j_amundsen      | Unrecognized VM option 'UseG1GC
+neo4j_amundsen      | Did you mean '(+/-)UseG1GC'?
+neo4j_amundsen      | Error: Could not create the Java Virtual Machine.
+neo4j_amundsen      | Error: A fatal exception has occurred. Program will exit.
+```
+**VS Code Note:** Use <kbd>ctrl</kbd><kbd>shift</kbd><kbd>p</kbd> to open `Command Palette`. Choose `LF`. Press `Enter`.
```
Comment (feng-tao) [inline docs/installation.md:99]: I kinda confused: any reason we put VS code image in the file? also is it an issue with windows or mac?

**Review by feng-tao — CHANGES_REQUESTED** [review]

File: docs/installation.md (hunk 1 above)
Comment (prafullkotecha) [reply docs/installation.md:99]: I experienced the issue with Windows. VS code image is linked from the installation document. I used VS Code to change the end-of-line sequence setting, from `CRLF` to `LF`. That's why I added it as a note.

**Comment by dorianj** [comment]
I think this is overly-specific for the main installation guide. If you wanted to start a Windows troubleshooting guide, would be happy to put this there.

**Comment by prafullkotecha** [comment]
@dorianj - How do I go about doing that?

**Comment by dorianj** [comment]
@prafullkotecha nothing too complicated -- just add a `windows_troubleshooting.md` (or whatever name) file, and then link it from the main instructions file

also, I forgot to mention -- thank you for submitting this! Glad to be able to support Windows

**Pull request closed without merging**